kong/runloop/handler.lua

```diff
@@ -408,6 +412,23 @@ return {
         balancer.init()
       end)
 
+      do
+        local err
+
+        -- semaphore (mutex) for build_api_router
+        build_api_router_semaphore, err = semaphore.new()
+        if err then
+          log(ERR, "failed to create build_api_router_semaphore: ", err)
+        end
+        build_api_router_semaphore:post(1)
+
+        -- semaphore (mutex) for build_router
+        build_router_semaphore, err = semaphore.new()
+        if err then
+          log(ERR< "failed to create build_router_semaphore: ", err)
```

Comment:
A typo here

Reply:
I am wondering if those logs should have a higher level than `error`, and I believe so. Probably `crit`, as they would make the system operate in what we would consider as critical conditions for its current lifecycle.

Reply:
yeah, CRIT definitely makes sense. Done :+1: